stable/tomcat/templates/_helpers.tpl

```diff
@@ -2,15 +2,15 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "tomcat.chart" -}}
+{{- define "tomcat.name" -}}
```

Comment:
You were going to refactor the chart based on `helm create` using Helm 2.8+. Are you still working on this?

Reply:
I just realized my helm client was outdated. I refactored the chart based Helm 2.9.0